server/ngwmn/templates/site_location.html

```diff
@@ -212,7 +212,11 @@ <h1>{{ feature.SITE_NAME }}</h1>
                         <tbody>
                             <tr>
                                 <td>{{ stats.overall.min_value }}</td>
+                                {% if stats.overall.median %}
+                                <td>{{ stats.overall.median }}</td>
+                                {% elif stats.overall.median_value %}
                                 <td>{{ stats.overall.median_value }}</td>
+                                {% endif %}
```

Comment:
We need to look into this as it turns out now seeing this in context. This is working when there is a value.